components/butterfly/src/server/expire.rs

```diff
@@ -12,92 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Expire suspected members.
-//!
-//! This module keeps track of suspected members, and sets their status to confirmed if they remain
-//! suspect long enough.
+//! Periodically check membership rumors to automatically "time out"
+//! `Suspect` rumors to `Confirmed`, and `Confirmed` rumors to
+//! `Departed`.
 
 use std::thread;
 use std::time::Duration;
 
-use time::SteadyTime;
-
-use member::Health;
 use rumor::{RumorKey, RumorType};
 use server::timing::Timing;
 use server::Server;
-use trace::TraceKind;
+
+const LOOP_PERIOD_MS: u64 = 500;
```

Comment:
I think it would be more accurate to call this `LOOP_DELAY_MS` since it always sleeps this amount irrespective of how long the rest of the loop takes to run rather than having the loop's runtime subtracted from the sleep.

Reply:
Yup, good point.